### ISSUE TITLE
ERROR: Compile command not found

### ISSUE THREAD

**Issue by zakabluk** [Backelite/sonar-swift#232]
After running run-sonar-swift.sh I got  Build Succeeded and then "Running SwiftLint...Linting Swift files at paths " had finished successfully. But next step had got error:

```
Skipping /Users/mac/sonarqueb-r/RBLU-iOS/BLUs/BLUs/.. Compile command not found.
...ERROR - Command 'oclint-json-compilation-database -v --include ../BLUs/. ./BLUs/. -- -rc LONG_LINE=250 -max-priority-1 10000 -max-priority-2 10000 -max-priority-3 10000 -report-type pmd -o sonar-reports/.-oclint.xml' failed with error code: 6  
```

**SonarQube environment:**
 - OS: Mac OS Mojave 10.14.5 
 - Version SonarQube Scanner 3.3.0.1492
- OClint 0.13

**Comment by davidy4ng** [comment]
You get an error 6 from OCLint, which according to their documentation means:

>6 - COMPILATION_ERRORS

https://oclint-docs.readthedocs.io/en/stable/manual/oclint.html

**Comment by gaelfoppolo** [comment]
This can be resolved using OClint 0.12.

**Comment by gaelfoppolo** [comment]
Closing this issue. Please open a new one if you're experiencing this.
